clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -418,30 +338,6 @@ public HostLevelIngestionStats(
         new Max(),
         TehutiUtils.getPercentileStat(getName() + AbstractVeniceStats.DELIMITER + storageEnginePutLatencySensorName));
 
-    this.produceToDrainerQueueCallCountSensor = registerPerStoreAndTotal(
```

Comment:
Would be great if we mention the list of dropped metrics in the commit description.  Hopefully there are not alerts on those.

Reply:
Good idea. I need to tweak the PR anyway for a spotbugs violation... so I'll also list the deleted metrics. Probably tomorrow.